ballerina/blocking_queue.bal

```diff
@@ -73,26 +71,18 @@ public class BlockingQueue {
     # ```
     # + item - The element to add to the queue.
     # + return - returns true upon success and throwing an IllegalStateException if no space is currently available
```

Comment:
This should be either true or Error. You may have to handle `IllegalStateException` from the native side and return a `BError`